package.json

```diff
@@ -1,7 +1,18 @@
 {
-  "name": "design-system-prototypes",
-  "private": true,
   "version": "0.0.0",
+  "author": "City of Amsterdam",
+  "description": "Prototyping environment for the Amsterdam Design System",
+  "license": "EUPL-1.2",
+  "name": "@amsterdam/design-system-prototypes",
```

Comment:
We're not going to publish this as a package or anything, right? Doesn't need to be in the @amsterdam namespace I think

Reply:
I hesitated already, it’s gone now.